src/server/api/routers/booking.ts

```
@@ -3,13 +3,15 @@ import { z } from "zod";
 
 import { logger } from "~/_utils";
 import { createTRPCRouter, protectedProcedure } from "~/server/api/trpc";
+import { captureEvent } from "~/server/_lib/posthog";
```

ESLint

[error] `~/server/_lib/posthog` import should occur before import of `~/server/api/trpc`

(import/order)